Help/MAC_Hints.rst

```diff
@@ -1,25 +1,74 @@
 Coverage Installation Hints for macOS Users:
 ============================================
 
+pyenv
+-----
+
+Here we use pyenv, which is simply a python installer that allows
+you to install multiple python versions and switch through them.
+In case your macOS has an older version of python that coala doesn't
+support we recommend using pyenv to install and shift to a newer
+supported version. After installation, you may also want to set up
+a virtual environment using :code:`venv` or :code:`virtualenv`.
```

Comment:
`pyenv` is a tool that solves a problem that originated because of the complexity of two python versions of the same machine. It's workflow is entirely different from conventional python tools.

I think just using homebrew to install/update python is fine, and does't create any additional complexity.